bootstrap/test/nerves_new_test.exs

```diff
@@ -59,4 +59,13 @@ defmodule Nerves.BootstrapTest do
     end
   end
 
+  test "new project cookie set", context do
```

Comment:
It would be nice to have a test that it generates a 64-character random cookie if you don't specify the option.